acceptance/testdata/auth/auth-switch.txtar

```diff
@@ -0,0 +1,58 @@
+# This requires a second account to log in to, meaning the GH_USER, 
+# GH_HOST_ALT, GH_USER_ALT, and GH_TOKEN_ALT env vars must be set.
+
+# Ensure that the GH_USER, GH_USER_ALT, and GH_TOKEN_ALT are available
+exec echo $GH_USER
+stdout '.+'
+
+exec echo $GH_HOST_ALT
+stdout '.+'
+
+exec echo $GH_USER_ALT
+stdout '.+'
+
+exec echo $GH_TOKEN_ALT
+stdout '.+'
+
+# Copy $GH_TOKEN to a new env var
+env LOGIN_TOKEN=$GH_TOKEN
+
+# Remove GH_TOKEN env var so we don't fall back to it
+env GH_TOKEN=''
+
+# Login to GH_HOST by feeding the GH_TOKEN to stdin
+exec echo $LOGIN_TOKEN
+stdin stdout
+exec gh auth login --hostname=$GH_HOST --with-token --insecure-storage
+
+# Confirm that the active user is GH_USER
+exec gh auth status --active
+stdout $GH_USER
+
+# Confirm that the active host is GH_HOST
+exec gh auth status --active
+stdout $GH_HOST
+
+# Login to GH_HOST_ALT by feeding the GH_TOKEN_ALT to stdin
+exec echo $GH_TOKEN_ALT
+stdin stdout
+exec gh auth login --hostname=$GH_HOST_ALT --with-token --insecure-storage
+
+# Confirm the active user is GH_USER_ALT
+exec gh auth status --active
+stdout $GH_USER_ALT
+
+# Confirm that the active host is GH_HOST_ALT
+exec gh auth status --active
+stdout $GH_HOST_ALT
+
+# Switch to the original user
+exec gh auth switch -h=$GH_HOST -u=$GH_USER
+
+# Confirm that the active user is GH_USER
+exec gh auth status --active
+stdout $GH_USER
+
+# Confirm that the active host is GH_HOST
+exec gh auth status --active
```

Comment:
I don't think this test is doing what you think it is. There's no such thing as an `--active` host. Here's my (redacted) output:

```
➜ gh auth status --active
github.com
  ✓ Logged in to github.com account williammartin (keyring)
  - Active account: true
  - Git operations protocol: https
  - Token: gho_************************************
  - Token scopes: 'delete_repo', 'gist', 'read:org', 'repo', 'workflow'

<other>
  ✓ Logged in to <other> account williammartin (keyring)
  - Active account: true
  - Git operations protocol: https
  - Token: gho_************************************
  - Token scopes: 'delete_repo', 'gist', 'read:org', 'repo', 'workflow'
```

So each host has an active user. If `GH_HOST` and `GH_HOST_ALT` are the same, there will be one entry but it will always be the same host. If they are different hosts there will be two entries but they will both be there. So an assertion for an active host can never fail.

Similarly, imagine that `GH_HOST` and `GH_HOST_ALT` are different, then the assertion on the active user can never fail because each host only has one user, and therefore that user is active and will appear in the output.

Reply:
Interesting... I was tinkering around with this with a very specific login state, which _may_ have allowed the behavior here to assert what I wanted it to, but that doesn't mean that it will be for all configurations of Hosts and Users, which would definitely result in flakiness... Good callout!

I think the behavior I actually want is `gh auth status --active -h $GH_HOST` and I need to remove the`GH_HOST_ALT` env var, essentially forcing this to be tested against two different user accounts on the same host.

That said, I'm not sure how valuable that is... 🤔

Reply:
Yes I think that's what I would expect. It's very valuable as a UX test for `auth switch` but a little less valuable for the GHEC data residency confidence since the command is entirely local.